bridges/ReutersBridge.php

```diff
@@ -0,0 +1,277 @@
+<?php
+class ReutersBridge extends BridgeAbstract
+{
+	const MAINTAINER = 'hollowleviathan, spraynard, csisoap';
+	const NAME = 'Reuters Bridge';
+	const URI = 'https://reuters.com/';
+	const CACHE_TIMEOUT = 1800; // 30min
+	const DESCRIPTION = 'Returns news from Reuters';
+
+	private $feedName = self::NAME;
+
+	/**
+	 * Wireitem types allowed in the final story output
+	 */
+	const ALLOWED_WIREITEM_TYPES = array(
+		'story',
+		'headlines'
+	);
+
+	/**
+	 * Wireitem template types allowed in the final story output
+	 */
+	const ALLOWED_TEMPLATE_TYPES = array(
+		'story'
+	);
+
+	const FEED_REGION_VALUE_US = 'us';
+	const FEED_REGION_VALUE_UK = 'uk';
+
+	const PARAMETERS = array(
+		'United Kingdom' => array(
+			'feed_uk' => array(
+				'name' => 'News Feed',
+				'type' => 'list',
+				'title' => 'Feeds from Reuters U.K edition',
+				'values' => array(
+					'Tech' => 'tech',
+					'Wire' => 'wire',
+					'Business' => 'business',
+					'World' => 'world',
+					'Politics' => 'politics',
+					'Science' => 'science',
+					'Energy' => 'energy',
+					'Aerospace and Defense' => 'aerospace',
+					'Special Reports' => 'special-reports',
+					'Top News' => 'home/topnews',
+					'Markets' => 'markets',
+					'Sports' => 'sports',
+					'UK' => 'uk',
+					'Entertainment' => 'entertainment',
+					'Environment' => 'environment',
+					'Lifestyle' => 'lifestyle',
+				)
+			)
+		),
+		'United States' => array(
+			'feed_us' => array(
+				'name' => 'News Feed',
+				'type' => 'list',
+				'title' => 'Feeds from Reuters U.S/International edition',
+				'values' => array(
+					'Tech' => 'tech',
+					'Wire' => 'wire',
+					'Health' => 'health',
+					'Business' => 'business',
+					'World' => 'world',
+					'Politics' => 'politics',
+					'Science' => 'science',
+					'Lifestyle' => 'life',
+					'Energy' => 'energy',
+					'Aerospace and Defense' => 'aerospace',
+					'Special Reports' => 'special-reports',
+					'China' => 'china',
+					'Top News' => 'home/topnews',
+					'Markets' => 'markets',
+					'Sports' => 'sports',
+					'USA News' => 'us',
+				)
+			)
```

Comment:
I suggest to extract common catogories to some array. Then use array_merge to add non-common ones.

Reply:
Hi, i tried to split up into common categories to some array and when i tried to merge, the compiler doesn't happy with me as const don't allow expression. Same happen with private. I don't know how to merge two array into one giant parameter as you saw in the code. There are spread operator which can use to merge but it's on PHP 7.4 which may failed some test.

I just hope this bridge will soon to be merged.